Python/libraries/recognizers-number-with-unit/recognizers_number_with_unit/number_with_unit/english/extractors.py

```diff
@@ -57,6 +61,10 @@ def __init__(self, culture_info: CultureInfo):
 # pylint: enable=abstract-method
 
 class EnglishAgeExtractorConfiguration(EnglishNumberWithUnitExtractorConfiguration):
+    @property
+    def ambiguity_filters_dict(self) -> Dict[Pattern, Pattern]:
```

Comment:
This property should be on the parent class. Same for all the other times it appears.